moml/simulation/quantum_mechanics/test_orca_simple.py

```
@@ -0,0 +1,145 @@
+def check_result_files(output_dir):
+    results_csv = output_dir / "orca_results_summary.csv"
+    if results_csv.exists():
+        try:
+            df = pd.read_csv(results_csv)
+            logger.info(f"Results entries: {len(df)}")
+        except Exception as e:
+            logger.error(f"Error reading results: {e}")
+    ml = output_dir / "ml_training_data.json"
+    if ml.exists():
+        try:
+            data = json.load(open(ml))
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)